res/xml/qwerty_ru.xml

```diff
@@ -0,0 +1,50 @@
+<?xml version="1.0" encoding="utf-8"?>
+<keyboard>
+  <row>
+    <key key0="й" key1="esc" key2="1" key3="~" key4="!"/>
+    <key key0="ц" key2="2" key3="\@"/>
+    <key key0="у" key2="3" key3="\#"/>
+    <key key0="к" key2="4" key3="$"/>
+    <key key0="е" key1="ё" key2="5" key3="%"/>
+    <key key0="н" key2="6" key3="^"/>
+    <key key0="г" key2="7" key3="&amp;"/>
+    <key key0="ш" key2="8" key3="*"/>
+    <key key0="щ" key1="accent_macron" key2="9" key3="(" key4=")"/>
```

Comment:
You don't need to include any accents, none of them will have any effects on Russian letters. The `accent_...` keys are only meant for the "programming" layouts, currently only `qwerty`.